### О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.

Формула нахождения дисперсии = (b-a)^2 / 12

Нам известно, что a = 0.5
Так же то, что d^2 = 0.2
Соответственно: (b - 0.5)^2 / 12

(b-0.5)^2 = квадрат разности, раскладывается, как b^2 - 2 * 0.5 * b + 0.5^2 = b^2 - b + 0.25

Составим уравнение: b^2 - b + 0.25 = 0.2 * 12

b^2 - b - 2.15 = 0

In [2]:
from math import sqrt

discriminant = 1 - (4 * 1 * (-2.15))
root = (1 + sqrt(discriminant)) / 2

root

2.049193338482967

Мы нашли b = 2.049 - Это и есть правая граница величины B

Подставим в формулу для проверки

In [3]:
((2.049193338482967 - 0.5) ** 2) / 12

0.20000000000000004

Теперь найдем среднее значение

M(X) = (a + b) / 2

In [4]:
(0.5 + root) / 2

1.2745966692414834

Ответ: правая граница величины B = 2.049, а среднее значение = 1.275